runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -167,4 +167,11 @@
   Boolean isShutdownSourcesOnFinalWatermark();
 
   void setShutdownSourcesOnFinalWatermark(Boolean shutdownOnFinalWatermark);
+
+  @Description(
+      "Interval in milliseconds for sending latency tracking marks from the sources to the sinks.")
+  @Default.Long(-1L)
+  Long getLatencyTrackingInterval();
```

Comment:
This looks like a global debug option and not a flink specific option. Lets move it to some thing like Tracking/PerformanceOption

Reply:
I do not follow. I do not think it is debug option. 
For reference: 
https://github.com/apache/flink/blob/master/flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java#L244

Where do you suggest it to move?

Reply:
I see, Sounds reasonable.